providers/k8s/connection/manifest/connection.go

```diff
@@ -123,11 +130,11 @@ func (c *Connection) AssetId() (string, error) {
 	// the same resource multiple times but it will result in different assets because of the random
 	// file name.
 
-	if len(c.Objects) == 1 {
+	if len(c.Objects) == 1 && c.asset.Platform.Runtime == "k8s-admission" {
```

Comment:
@imilchev Please double-check.

This was also broken in v8. When we have a manifest file with only one object in it, we would use the uid. But as I understand the comment, this was only meant to be used with Admission scans.

v8:
```
  ids: [
    0: "//platformid.api.mondoo.app/runtime/k8s/uid/3eae16c2-2b55-4048-a84b-03a849d12e40"
  ]
```

v9 with this PR:
```
  ids: [
    0: "//platformid.api.mondoo.app/runtime/k8s/uid/044dbf3db4656a3863d97901ab262e11d184641014d4cb10bf6d57c78da8f437"
  ]
```

Reply:
your change is the correct behaviour here! nice catch!